### ISSUE TITLE
Add option to change text color

### ISSUE THREAD

**Issue by ZebulanStanphill** [WordPress/twentytwenty#3]
One thing you can't easily do with Twenty Twenty right now is use a dark color scheme with light-colored text. You can change the background color, but you can't change the text color, so you're effectively limited to choosing light background colors to preserve text contrast. I think it would be a good idea to add a text color option or at least a dark/light text toggle so people can make dark-themed websites.

**Comment by carolinan** [comment]
We need to consider this carefully in combination with https://github.com/WordPress/twentytwenty/issues/40, _Theme background colour should be used in the editor._

Instead of a toggle or text color option, we could calculate when the text color needs to be adjusted, and when to enable the editor dark mode.

**Comment by celloexpressions** [comment]
I would also advocate for auto-updating the text color based on the background color. This is a relatively straightforward matter of calculating the relative luminance of the user-selected color and setting a theme mod that toggles a body class accordingly. This slight amount of technical debt creates a significantly-enhanced user experience following "decisions, not options" and other aspects of the WordPress philosophy. (The editor should have also taken this approach, but didn't)

The implementation could probably be done primarily via customize-controls JS and a small amount of customize-preview JS to instantly update as users select a color. I have also had success with a more-involved approach that leverages a PHP-side color patterns file, see 
https://celloexpressions.com/blog/a-strategy-for-custom-colors-in-the-customizer/

Are there any other user-customizable colors that would benefit from an enhanced color handling strategy with automated contrast adjustments?

**Comment by carolinan** [comment]
Closing in favor of https://github.com/WordPress/twentytwenty/issues/284
